frontera/contrib/backends/sqlalchemy/components.py

```diff
@@ -79,7 +83,7 @@ def _modify_page(self, obj):
     def _create_page(self, obj):
         db_page = self.model()
         db_page.fingerprint = obj.meta['fingerprint']
-        db_page.url = obj.url
+        db_page.url = obj.url_as_unicode
```

Comment:
why it's converted here to unicode? May be we discussed that, but I please help me to remember the reasoning.